scripts/azure_set_gate_variable_cloud_account_id.ps1

```diff
@@ -45,5 +45,6 @@ if (Test-Path $path) {
     Write-Host "##vso[task.setvariable variable=Sku;isOutput=true]$sku"
     Write-host "The value of Variable IsEnabled is updated to True and output variable ImageUrl to $uri"
 } else {
-    Write-Host "Artifact path $path does NOT exist for $Version"
+    #Write-Host "Artifact path $path does NOT exist for $Version"
+    throw "Artifact path does NOT exist for $Version" # Throwing error if there's no baseimage, instead of just writing it to the host.
```

Comment:
This is good coding. But remove all the comments - line 48 entirely and the comment on the end of line 49. It doesn't add anything. Its standard POSH coding at LANSA.

The same for the other script.